### PR TITLE
Display fee values - txp details

### PR DESCRIPTION
**Pull request by JDonadio**

### CODE REVIEW

File: src/js/controllers/modals/txpDetails.js [hunk 1]
```diff
@@ -16,11 +16,21 @@ angular.module('copayApp.controllers').controller('txpDetailsController', functi
...
     applyButtonText();
   };
 
+  function displayFeeValues() {
+    txFormatService.formatAlternativeStr($scope.tx.fee, function(v) {
+      $scope.tx.feeFiatStr = v;
+    });
+    $scope.tx.feeRateStr = ($scope.tx.fee / ($scope.tx.amount + $scope.tx.fee) * 100).toFixed(2) + '%';
+    if ($scope.tx.feeLevel == 'superEconomy') $scope.tx.feeLevelStr = gettextCatalog.getString('Super Economy');
+    else $scope.tx.feeLevelStr = $scope.tx.feeLevelStr = $scope.tx.feeLevel.charAt(0).toUpperCase() + $scope.tx.feeLevel.slice(1);
```
Comment (gabrielbazan7) [inline src/js/controllers/modals/txpDetails.js]: $scope.tx.feeLevelStr is repeated.

File: src/js/controllers/modals/txpDetails.js [hunk 2]
```diff
@@ -16,11 +16,21 @@ angular.module('copayApp.controllers').controller('txpDetailsController', functi
...
     checkPaypro();
     applyButtonText();
   };
 
+  function displayFeeValues() {
+    txFormatService.formatAlternativeStr($scope.tx.fee, function(v) {
+      $scope.tx.feeFiatStr = v;
+    });
+    $scope.tx.feeRateStr = ($scope.tx.fee / ($scope.tx.amount + $scope.tx.fee) * 100).toFixed(2) + '%';
+    if ($scope.tx.feeLevel == 'superEconomy') $scope.tx.feeLevelStr = gettextCatalog.getString('Super Economy');
```
Comment (gabrielbazan7) [inline src/js/controllers/modals/txpDetails.js]: Should be:  $scope.tx.feeLevelStr = feeService.feeOpts[$scope.tx.feeLevel]

Comment (JDonadio) [inline src/js/controllers/modals/txpDetails.js]: Awesome!

**Comment by JDonadio** [comment]
Updated. Thanks!

File: www/views/tx-details.html [hunk 3]
```diff
@@ -77,11 +77,11 @@
           </div>
           <i class="icon bp-arrow-right"></i>
         </a>
-        <div class="item single-line" ng-click="toggleFeeFiat = !toggleFeeFiat" ng-if="btx.action != 'received'">
```
Comment (gabrielbazan7) [inline www/views/tx-details.html]: This is ok ? Fees should only show on sent transactions

Comment (JDonadio) [inline www/views/tx-details.html]: Excellent. thanks!

**Pull request merged**